DateTimeOnly/packages.lock.json

```diff
@@ -34,6 +43,11 @@
         "resolved": "1.1.1",
         "contentHash": "AT3HlgTjsqHnWpBHSNeR0KxbLZD7bztlZVj7I8vgeYG9SYqbeFGh0TM/KVtC6fg53nrWHl3VfZFvb5BiQFcY6Q=="
       },
+      "Microsoft.NETFramework.ReferenceAssemblies.net462": {
```

Comment:
The same is true for this one - we don't need this dependency here.

Reply:
Removed this reference too.